roles/pre_tasks/README.md

```diff
@@ -38,12 +38,30 @@ tower_setup_file: ansible-tower-setup-{{ tower_release_version }}.tar.gz
 tower_hosts:
   - "localhost ansible_connection=local"
 
-tower_database: ""
+tower_database_host: ""
```

Comment:
So I noticed an issue, We have these variables in both pre_tasks and in install, and in install_cop because they are all interconnected. While its fine to just change the vars in pretasks, the Readme's need to be updated in all 3 places so we can keep things straight. Is there a better way to do this? or should we just link to the pre tasks readme in the other 2 places? Not sure on what's the best answer, leaning towards keeping it in all 3, but only slightly. What are other peoples thoughts?

Reply:
I have the impression that it's only an impression :wink: 

The following command shows that `tower_database[...whatever...]` is documented almost everywhere, but only used in "pre_tasks":

```
tower_cert/README.md
backup/README.md
install/README.md
pre_tasks/defaults/main.yml
pre_tasks/tasks/main.yml
pre_tasks/templates/inventory.j2
pre_tasks/README.md
restore/README.md
```

I strongly suspect a wild cut&paste effect. We should do the same exercise of cleaning up README.md resp. `defaults/main.yml` as I did for the tower_configuration.

Nevertheless, I had overseen a few places to renamed `tower_database` to `tower_database_host` but I'll create a separate issue for the clean-up, if it's OK for you.

Reply:
It may be useful creating a doc file of the shared variables and importing them into each readme to save duplication. Not if you can do an include in markdown. Could be worth switching the readme to asciidoc as you can definitely do imports in that and it will still render in GitHub

Reply:
What about having the discussion in the issue #32 I created so that this PR can be merge?

Reply:
Yep we'll do it there, and merge.